### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by pchanda2541**
Change https://github.com/SNP-FIT/snp-light to https://github.com/SNP-FIT/snp-sensor

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @pchanda2541. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/06/13 15:21:36 Loaded 7060 libraries from DB
Changing URL of library SNP_Sensor from https://github.com/SNP-FIT/snp-light.git to https://github.com/SNP-FIT/snp-sensor.git
Success!
```

**Pull request merged**
